src/menu/chapter-thumbnail-menu-button.js

```diff
@@ -162,27 +142,38 @@ class ChapterThumbnailMenuButton extends VjsMenuButton {
     return items;
   }
 
-  /**
-   * @name Update
-   * @description
-   * Defined by videojs.MenuButton
-   *
-   * Must override so that the menu isn't attached to the videojs.MenuButton.
-   * Instead we will manage it ourselves by adding it to the player directly.
-   */
+  findChaptersTrack() {
+    const tracks = this.player().remoteTextTracks() || [];
 
-  update() {
-    const menu = this.createMenu();
+    for (let i = 0, length = tracks.length; i < length; i++) {
+      const track = tracks[i];
 
-    this.menu = menu;
+      if (track.id === TRACK_ID) {
+        return track;
+      }
+    }
 
-    this.buttonPressed_ = false;
+    return undefined;
+  }
 
-    if (this.items && this.items.length === 0) {
-      this.hide();
-    } else if (this.items && this.items.length > 1) {
-      this.show();
+  setTrack(track) {
+    if (!track) {
+      return;
+    }
+
+    this.track = track;
+
+    this.track.mode = 'hidden';
+
+    const remoteTextTrackEl = this.player().remoteTextTrackEls().getTrackElementByTrack_(
+      this.track
+    );
+
+    if (!remoteTextTrackEl) {
+      return;
     }
+
+    remoteTextTrackEl.addEventListener('load', () => this.update());
```

Comment:
this method is called by `video.js::MenuButton`  or such that will recreate the menu and the items—or something…